oscn/parse/attorneys.py

```diff
@@ -0,0 +1,20 @@
+from bs4 import BeautifulSoup
+
+
+def attorneys(oscn_html):
+    attorney_list = []
+    soup = BeautifulSoup(oscn_html, 'html.parser')
+    start = soup.find('h2', 'section attorneys')
+    sibling = start.next_sibling.next_sibling
+    if sibling.name == "table":
+        attorney_table = sibling
+        rows = attorney_table.find('tbody').find_all('tr')
+        for row in rows:
+            attorney = row.td.contents[0]
```

Comment:
I suggest using `row.td.text` to strip html and avoid the need to use split.

Reply:
The td actually includes the name and address of the attorney, and it comes out pretty ugly. I just pulled out the name, and used `contents`, because that made it prettiest. The split is actually cleaning up a bunch of leftover white space, which you still get with row.td.text, so I would still keep the line with split.

But do we want the address? Not every one comes with an address, but we could potentially include it. Or we could easily include it later. It's in the list created by `contents`.

Reply:
I'm not sure what we will do with this.  Maybe later we will have 'sub-parsers' when we decide.  For now I'd like to be consistent with `.text` because it is smaller.

Reply:
Not sure what you mean. Shorter by 5 characters? Not sure why that matters or if that's even an advantage.

This is the output from this PR:
`Collier, Mark (Bar #13260)`

This is the output if we replace `contents[0]` with `text`
`Collier, Mark (Bar #13260)COLLIER & HUBER LAW FIRM PC616 S BOSTON AVER., SUITE 308TULSA, OK 74119`

Note that there's not even a space between the name and the address. Very ugly. Both above are still using the split line, because otherwise there is a bunch of excess whitespace and at least one carriage return.